app/upgrades/v1/bank_send_list.go

```diff
@@ -0,0 +1,65 @@
+package v1
+
+import (
+	"encoding/json"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func BankSendList(ctx sdk.Context) (ResultList, error) {
+	ctx.Logger().Info(fmt.Sprintf("bank send list:%s", UpgradeName))
+
+	// Read file and get list
+	var result ResultList
+	json.Unmarshal([]byte(BANK_SEND_LIST), &result)
+
+	return result, nil
+}
+
+const BANK_SEND_LIST string = `{
+	"validator": [
+		{
+			"fromAddress": "ununifi132ap8qzhmzn9edyjzz290xvr96dgzp2khhapk7",
```

Comment:
Isn't it better that you separate`BankSendTarget` from `fromAddress` because there're many `BankSendTarget` despite of a few `fromAddress`, is it?

Reply:
ok